database/seeders/MembershipBannerSeeder.php

```diff
@@ -6,6 +6,7 @@
 
 class MembershipBannerSeeder extends Seeder
 {
+    // XXX Can we get rid of this seeder and use factories to generate these when needed?
```

Comment:
Yes, I think so. This was a point-in-time change as well, not one we'll need to do again. See more context in WEB-596.